cypress/plugins/index.js

```diff
@@ -18,4 +18,45 @@
 module.exports = (on, config) => {
   // `on` is used to hook into various events Cypress emits
   // `config` is the resolved Cypress config
+  config.process = {
```

Comment:
do we really need to redefine all process variables like this or can we do this in a more targeted fashion? Also we need some form of documentation of why we started doing this and what its supposed to achieve. Again links to open issues would be a great help here.

Reply:
It looks like my rebase wasn't perfect after all, I'll fix that.